plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -1,15 +1,59 @@
-const adminEmail =
-	process.env.USE_WP_ENV === '1'
-		? 'wordpress@example.com'
-		: 'admin@woocommercecoree2etestsuite.com';
+const {
+	ADMIN_USER,
+	ADMIN_PASSWORD,
+	ADMIN_USER_EMAIL,
+	CUSTOMER_USER,
+	CUSTOMER_PASSWORD,
+	CUSTOMER_USER_EMAIL,
+	USE_WP_ENV,
+} = process.env;
+
+const admin = {
+	username: ADMIN_USER ?? 'admin',
+	password: ADMIN_PASSWORD ?? 'password',
+	email:
+		ADMIN_USER_EMAIL ??
+		( !! USE_WP_ENV
+			? 'wordpress@example.com'
+			: 'admin@woocommercecoree2etestsuite.com' ),
+};
+
+const customer = {
+	username: CUSTOMER_USER ?? 'customer',
+	password: CUSTOMER_PASSWORD ?? 'password',
+	email: CUSTOMER_USER_EMAIL ?? 'customer@woocommercecoree2etestsuite.com',
+	billing: {
+		us: {
+			first_name: 'Maggie',
+			last_name: 'Simpson',
+			address: '123 Evergreen Terrace',
+			city: 'Springfield',
+			country: 'US',
+			state: 'OR',
+			zip: '97403',
+			phone: '555 555-5555',
+			email: 'customer@example.com',
+		},
+		malta: {
+			first_name: 'Maggie',
+			last_name: 'Simpson',
+			address: '123 Evergreen Terrace',
+			city: 'Valletta',
+			country: 'MT',
+			zip: 'VT 1011',
+			phone: '555 555-5555',
+			email: 'vt-customer@example.com',
+		},
+	},
+};
```

Comment:
Instead of using a lot of environment variables like ADMIN_USER, ADMIN_PASSWORD, etc. inside individual spec files, I added admin and customer objects in test-data/data.js that tests can use to retrieve their username, password, billing, and other details.

Related to my comment in:
https://github.com/woocommerce/woocommerce/pull/35492/files#r1013814136